src/amo/components/OverallRating.js

```diff
@@ -1,37 +1,69 @@
 import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
 
+import { setUserRating } from 'amo/actions/ratingActions';
+import { callApi } from 'core/api';
 import translate from 'core/i18n/translate';
+import log from 'core/logger';
 
 import 'amo/css/OverallRating.scss';
 
 
 export class OverallRatingBase extends React.Component {
   static propTypes = {
-    i18n: PropTypes.object,
-    addonName: PropTypes.string,
+    addonName: PropTypes.string.isRequired,
+    addonID: PropTypes.number.isRequired,
+    authToken: PropTypes.string,
+    version: PropTypes.object.isRequired,
+    i18n: PropTypes.object.isRequired,
+    createRating: PropTypes.func.isRequired,
+  }
+
+  onChange = ({ currentTarget: form }) => {
+    const input = form.querySelector('input:checked');
+    if (!input) {
+      // I don't think any user input would cause a form change like
+      // this. I'm not sure what would cause this at all but it's probably
+      // better not to throw an exception.
+      log.error('The form changed but no input was selected');
+    } else {
+      log.debug('Selected rating from form input:', input.value);
+      this.props.createRating({
+        rating: parseInt(input.value, 10),
+        versionID: this.props.version.id,
+        authToken: this.props.authToken,
+        addonID: this.props.addonID,
+      });
+    }
   }
 
   render() {
-    const { i18n, addonName } = this.props;
+    const { i18n, addonName, version } = this.props;
     const prompt = i18n.sprintf(
-      i18n.gettext('How are you enjoying your experience with %(addonName)s?'),
-      { addonName });
+      i18n.gettext(
+        'How are you enjoying your experience with ' +
+        '%(addonName)s %(version)s?'),
```

Comment:
I feel like most users wouldn't understand the difference. Some add-ons would look like: `How are you enjoying your experience with Ad Blocker 3.5-20160920192015?` which would likely just be confusing.

Opening a UX issue sounds good to me but rating a version rather than an add-on feels like an implementation detail to me. As far as the user is concerned they rated the add-on. I'd say leave the string change out unless UX wants it.